src/Sylius/Bundle/PromotionBundle/test/src/Entity/PromotionSubject.php

```diff
@@ -11,15 +11,21 @@
 
 declare(strict_types=1);
 
-namespace AppBundle\Entity;
+namespace App\Entity;
 
 use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
 use Sylius\Component\Promotion\Model\PromotionInterface;
 use Sylius\Component\Promotion\Model\PromotionSubjectInterface;
 use Sylius\Resource\Model\ResourceInterface;
 
+#[ORM\Entity]
+#[ORM\Table(name: 'app_promotion_subject')]
 class PromotionSubject implements ResourceInterface, PromotionSubjectInterface
 {
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
     private int $id;
```

Comment:
Is it a good time to update this Entity as well? No constructor with ArrayCollection assign looks broken.


```suggestion
    private ?int $id;
```
and

```patch
    /** @var Collection<array-key, PromotionInterface> */
---    protected $promotions;
+++    protected Collection $promotions;
```
and

```patch
+++    public __construct()
+++    {
+++        $this->promotions = new ArrayCollection();
+++    }
```